lib/Basics/Result.h

```diff
@@ -23,7 +23,8 @@
 #ifndef ARANGODB_BASICS_RESULT_H
 #define ARANGODB_BASICS_RESULT_H 1
 
-#include "Basics/Common.h"
+#include <string>
+#include "Basics/voc-errors.h"
```

Comment:
can't we just include `Basics/Common.h` here?

Reply:
The result is used in ` Basics/win-utils.h` which is part of `Basics/Common.h` and therefore it would result in a cyclic dependency. How about adding the Result to Common.h?

Reply:
Fine for me.